### PR TITLE
Closes #213 [Description on the first web page]

### LINKED ISSUE #213: Description on the first web page
The first page is important. We want the user to get a clear understanding of what the tool can do and a positive assessment of its value. Background is much less important. 

"open AI-assisted mapping service":  Here "open" probably is intended to say FOSS. But the service is _not_ FOSS, although the SW that it is based on is. "AI-assisted mapping service" is an overly convoluted way of saying that it performs mapping. 

"roads, waterways, and trees": This is false. Hopefully it will be true, but right now it is not.

"specifically computer vision techniques": provides almost no information. 

"satellite and UAV imagery": fAIr does not support use of satellite imagery

"f: for freedom and free and open-source software": "freedom" is not informative, FOSS is very important to me, but still rather background than central information about fAIr. 

"r: for resilience and our responsibility for our communities and the role we play within humanitarian mapping": Provides very close to no information at all about fAIr.

"fAIr is an open AI-assisted mapping service developed by the Humanitarian OpenStreetMap Team (HOT) that aims to improve the efficiency and accuracy of mapping efforts for humanitarian purposes. The service uses AI models, specifically computer vision techniques, to detect objects such as buildings, roads, waterways, and trees from satellite and UAV imagery. The name fAIr is derived from the following terms:

f: for freedom and free and open-source software
AI: for Artificial Intelligence
r: for resilience and our responsibility for our communities and the role we play within humanitarian mapping

Suggested replacement:

fAIr does mapping in the same way as human mappers using HOTs Tasking Manager. It looks at UAV imagery and produces map data that can be added to the OSM. Tests show a 100% speedup compared to manual mapping. It uses Artificial Intelligence (AI) to accomplish this. 

fAIr is developed by the Humanitarian OpenStreetMap Team and all the software is free and open source.

Before fAIr is used it needs to be fine-tuned by training on high quality map data for a small representative part of the geographical region where it is to be used.

### PR DESCRIPTION
**Pull request by natrimmer**
This PR addresses the home page text concerns raised by Issue #213. I updated the text using the suggested changes.

### CODE REVIEW

File: frontend/src/components/Layout/Home/Home.js [hunk 1]
```diff
@@ -26,23 +26,13 @@ const GetStarted = () => {
...
         style={{ color: "#3D3D3D", fontSize: "18px", marginBottom: "50px" }}
       >
-        fAIr is an open AI-assisted mapping service developed by the
-        Humanitarian OpenStreetMap Team (HOT) that aims to improve the
-        efficiency and accuracy of mapping efforts for humanitarian purposes.
-        The service uses AI models, specifically computer vision techniques, to
-        detect objects such as buildings, roads, waterways, and trees from
-        satellite and UAV imagery. The name fAIr is derived from the following
-        terms:
+        fAIr performs mapping in the same way as human mappers using HOT's Tasking Manager. It looks at UAV imagery and produces map data that can be added to OpenStreetMap (OSM). Tests show a 100% speedup compared to manual mapping. It uses Artificial Intelligence (AI) to accomplish this.
```
Comment (kshitijrajsharma) [inline frontend/src/components/Layout/Home/Home.js:29]: I am little bit curious of 100 % number , Is it coming from some analysis ? Just curious

**Review by kshitijrajsharma — APPROVED** [review]

File: frontend/src/components/Layout/Home/Home.js (hunk 1 above)
Comment (omranlm) [reply frontend/src/components/Layout/Home/Home.js:29]: I think the 100% sppedup is coming from [this article](https://www.hotosm.org/tech-blog/hot-tech-talks-fair/) Why fAIr? section

based on the average 1000-1500 buildings mapped per working day without AI assistance and the pilot when the average mapping nearly doubled to 2500-3000.

@natrimmer, confirm? 
or do you have other sources?

Comment (natrimmer) [reply frontend/src/components/Layout/Home/Home.js:29]: Yes that data point from @Ettrig 's suggested copy replacement does seem to come from that blog post.

Comment (kshitijrajsharma) [reply frontend/src/components/Layout/Home/Home.js:29]: Okay thanks both

**Pull request merged**